core/src/main/java/org/apache/carbondata/core/metadata/blocklet/BlockletInfo.java

```diff
@@ -261,7 +262,8 @@ private void writeChunkInfoForOlderVersions(DataOutput output) throws IOExceptio
 
   private DataChunk deserializeDataChunk(byte[] bytes) throws IOException {
     ByteArrayInputStream stream = new ByteArrayInputStream(bytes);
-    ObjectInputStream inputStream = new ObjectInputStream(stream);
+    ObjectInputStream inputStream =
```

Comment:
this method seems that "inputStream.close();" doesn't use finally block,  can you add the protection in this pr

Reply:
It does not open the file to mandatorily close the stream, it just opens on the byte[] so it is not needed to close in finally